### PR TITLE
feat: add Tour details view

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#75345

### CODE REVIEW

**Comment by gca-axelor** [comment]
http://75346-preview.cloud-sw1.axelor.io/

File: packages/apps/crm/src/api/index.ts [hunk 1]
```diff
@@ -74,5 +74,11 @@ export {
   updateProspect as updateProspectApi,
   updateProspectScoring,
 } from './prospect-api';
-export {searchTour as searchTourApi} from './tour-api';
-export {searchTourLine as searchTourLineApi} from './tour-line-api';
+export {
+  searchTour as searchTourApi,
+  validateTour as validateTourApi,
+} from './tour-api';
```
Comment (vhu-axelor) [inline packages/apps/crm/src/api/index.ts:80]: Add fetchTourById

**Review by vhu-axelor — CHANGES_REQUESTED** [review]

**Review by vhu-axelor — APPROVED** [review]
Nice work! 🤩

File: packages/apps/crm/src/screens/tour/TourListScreen.js [hunk 2]
```diff
@@ -91,8 +91,13 @@ const TourListScreen = ({navigation}) => {
           <TourCard
             name={item.name}
             date={item.date}
-            tourId={item.id}
+            tour={item}
             salesperson={item.salespersonUser?.fullName}
```
Comment (lme-axelor) [inline packages/apps/crm/src/screens/tour/TourListScreen.js:95]: Only tour prop is needed here

File: packages/apps/crm/src/components/molecules/TourValidateButton/TourValidateButton.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import {Button, useThemeColor} from '@axelor/aos-mobile-ui';
+import {useTranslator, useDispatch} from '@axelor/aos-mobile-core';
+import {validateTour} from '../../../features/tourSlice';
+
+const TourValidateButton = ({tourId}: {tourId: number}) => {
+  const I18n = useTranslator();
+  const Colors = useThemeColor();
+  const dispatch = useDispatch();
+
+  return (
```
Comment (lme-axelor) [inline packages/apps/crm/src/components/molecules/TourValidateButton/TourValidateButton.tsx:29]: This button should be visible only if a line is not validated on the tour

**Review by vhu-axelor — APPROVED** [review]

**Pull request merged**